apps/frontend/src/app/Database/DataManagers/CharacterData.ts

```diff
@@ -333,6 +333,9 @@ export class CharacterDataManager extends DataManager<
         else this.set(c.key, c)
       })
 
+      // Somnia shouldn't be removed on import.
+      idsToRemove.delete('Somnia')
```

Comment:
Her artifacts still get nuked on scan import. Maybe intended, since the scanner reports them as inventory, and it would mess with use-equipped?

Reply:
oh yeah that logic is very tricky, that might get resolved somewhat by loadouts, I'll see if i can figure out the import logic on equipment

Reply:
Also realized that the weapon needs to be added as exception